crates/tuono/src/watch.rs

```diff
@@ -3,8 +3,8 @@ use watchexec_supervisor::command::{Command, Program};
 
 use miette::{IntoDiagnostic, Result};
 use watchexec::Watchexec;
-use watchexec_signals::Signal;
 use watchexec_supervisor::job::{start_job, Job};
+use watchexec_supervisor::Signal;
```

Comment:
Why this change? Technically, `watchexec_supervisor::Signal` is just a re-export from `watchexec_signals::Signal`

Reply:
You're right - what happened was I updated `watchexec` to 5.0.0 first (while `watchexec_signals` was still on 3.0.0) and encountered the error mentioned in the PR description. Changing the import to `watchexec_supervisor::Signal` fixed it. But updating `watchexec_signals` to 4.0.0 fixes the error as well since `watchexec` 5.0.0 is dependent on it. 

I'll revert the import back to `watchexec_signals::Signal` in a later commit as it's no longer necessary